lessons/lesson-2/tasks/task-6.js

```diff
@@ -0,0 +1,7 @@
+// Реализовать функцию для перемешивания элементов массива
+
+function shuffle(array) {
+	return array.map((item) => ({ value: item, sort: Math.random() }))
+		.sort((a, b) => (a.sort - b.sort))
+		.map((item) => item['value'])
+}
```

Comment:
Метод `Array.sort()` выполняет один цикл сопоставления пар элементов массива, таким образом мы можем не сохранять порядок сортировки перед его вызовом, а полностью отдаться на волю случая 🙂

```suggestion
function shuffle(array) {
	return [...array].sort(() => 0.5 - Math.random())
}
```

Reply:
@PonomareVlad Исправил.